### PR TITLE
Update static config

### PR DESCRIPTION
**Pull request by DaintyJet**
## Overview of Changes

Updated Linux Static Configuration Documents (standalone) and the Harvester Document to refer to routes and DNS configuration as creating internal networks often requires this.

## Reason for Modification

This cleans up some of the old documentation (marginally) and it updates the new Harvester docs with additional information that may be helpful.

## Verification

- [x] The documentation is completed, or does not contain work-in-progress/partially-completed sections.
- [x] The documentation is written in *Markdown* (Exception for Inject and Incident Reports).
- [x] Any Images are contained in a subdirectory `Images`.
- [ ] Any scripts, Ansible Playbooks, Terraform scripts, etc. Have documentation explaining their purpose and use case.
- [ ] Any scripts are tested.
- [x] I have assigned and notified a reviewer.

### CODE REVIEW

**Comment by DaintyJet** [comment]
Please DO NOT merge till others I have requested reviewed this.

**Review by ryanwiddop — COMMENTED** [review]
This looks good to me

File: Admin/Infrastructure/Harvister/Network Creation/README.md [hunk 1]
```diff
@@ -13,6 +18,9 @@ Click Create and fill out the information in the *Basics* window. If you chose t
 
 <img src="Images/VMC.png">
 
+> [!IMPORTANT]
+> Selecting the *Untagged* network is preferred to prevent any internal conflicts, and we have observed it is much more stable in our environment.
+
 In the *Route* window configure it to use the **Manual** mode so we can make this network *internal* and not able to immediately access the internet. Fill in a `CIDR` range that does not conflict with the host's network (In our case do not use addresses that conflict with `192.168.0.0/21`). You should also set the `Gateway` to be some address in the `CIDR` range you selected.
```
Comment (P-rple) [inline Admin/Infrastructure/Harvister/Network Creation/README.md:24]: I would add, how can someone determine and confirm the host's network CIDR range?
For the purpose of longevity of this document, in the event it ever changes.

File: Admin/Infrastructure/Harvister/Network Creation/README.md [hunk 2]
```diff
@@ -64,6 +72,15 @@ Use the `ip` command to list the interfaces on your machine. We will need the *M
     <img src="Images/IPLINK.png">
 
 
+> [!NOTE]
+> The use of the following options is **OPTIONAL** you can remove all of them, and the netplan can be applied to multiple VMs without issues.
+> ```
+> match:
+>     macaddress: MACADDR
```
Comment (P-rple) [inline Admin/Infrastructure/Harvister/Network Creation/README.md:79]: Make a note that "MACADDR" is a placeholder for an actual mac address
"FF:FF:FF:FF:FF:FF"

**Review by P-rple — CHANGES_REQUESTED** [review]
I've added some simple suggestions.

File: OperatingSystem-Services/Platform-Linux/Static-Config/README.md [hunk 3]
```diff
@@ -1,133 +1,149 @@
...
 
   ![Alt text](Images/NMTUI-3.png)
 
-5. Move over to *Show* and hit enter
+5. Move over to *Show* and hit enter.
 
   ![Alt text](Images/NMTUI-4.png)
 
-6. Click *add* and fill out the information (IP address *We are on a /21 network*)
+1. Click *add* and fill out the information.
```
Comment (P-rple) [inline OperatingSystem-Services/Platform-Linux/Static-Config/README.md:43]: Is this supposed to be 6. instead of 1.

**Review by P-rple — CHANGES_REQUESTED** [review]
Fix number bullet point

**Review by P-rple — APPROVED** [review]

**Pull request merged**